modules/plink/vcf/main.nf

```diff
@@ -0,0 +1,44 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process PLINK_VCF {
+    tag "$meta.id"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::plink=1.90b6.21" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/plink:1.90b6.21--h779adbc_1"
+    } else {
+        container "quay.io/biocontainers/plink:1.90b6.21--h779adbc_1"
+    }
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*.bed"), emit: bed, optional: true
```

Comment:
Shouldn't the outputs be compressed?

Reply:
Not sure about that, actually we've going to consume this module very soon in gwas pipeline, if there's a need for enhancement we'd be happy to push those suggestions here as well and ask for your review 🙏 

For the time being, I hope it's okay if we go ahead with the merge and then work on the usage of this module.